pkg/dogstatsd/parse.go

```diff
@@ -22,10 +23,25 @@ var (
 	fieldSeparator = []byte("|")
 	colonSeparator = []byte(":")
 	commaSeparator = []byte(",")
-
-	commaSeparatorString = ","
 )
 
+// parser parses dogstatsd messages
+// not safe for concurent use
+type parser struct {
+	interner *stringInterner
+}
+
+func newParser() *parser {
+	stringInternerCacheSize := config.Datadog.GetInt("dogstatsd_string_interner_size")
```

Comment:
@remeh just like other config options, this should be defined in `config/config.go` and should have a default. The default may be left to `0` on `config.go` defined in the present file if it makes more sense.

Reply:
Whoops I'm pushing a PR for this (+ `aggregator_buffer_size` recently merged)...

Reply:
For history: default values properly set here: https://github.com/DataDog/datadog-agent/pull/4893